cmd/utils/flags.go

```diff
@@ -122,6 +122,14 @@ var (
 		Name:  "disablesnapprotocol",
 		Usage: "Disable snap protocol",
 	}
+	DisableDiffProtocolFlag = cli.BoolFlag{
+		Name: "disablediffprotocol",
+		Usage: "Disable diff protocol",
```

Comment:
Please add more description.  When should user enable or disable it , what is the default option.